main.cpp

```diff
@@ -1,14 +1,26 @@
-#include <iostream>
 #include "i.h"
 #include "u.h"
+#include <iostream>
 
 using namespace std;
 using i32 = mgt::i<32>;
+using u32 = mgt::u<32>;
 
 int main() {
   i32 a, b;
-  cin >> a >> b;
-  cout << (a < b) << endl;
+  bool flag = true;
```

Comment:
avoid use variable name like `flag`, in which you cannot get any useful information.